cms/templates/cms/faqs_page.html

```diff
@@ -0,0 +1,20 @@
+{% extends "cms/program_page.html" %}
+
+{% block social %}
+{% endblock %}
+
+{% block panel %}
+  <div class="faqs">
+    {% for faq in faqpage.faqs.all %}
+    <div class="faq">
+      <div class="question">
+        {{ faq.question }}
+      </div>
+      <div class="answer">
+        {{ faq.answer|safe }}
```

Comment:
This isn't a good idea. If you're intending to allow rich text use the `richtext` filter instead: http://docs.wagtail.io/en/v1.6.3/topics/writing_templates.html#rich-text-filter